graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/migrations/V20170607164210_MigrateReopenedIndicesToAliasesClusterStateES7.java

```diff
@@ -0,0 +1,35 @@
+package org.graylog.storage.elasticsearch7.migrations;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.graylog.shaded.elasticsearch7.org.elasticsearch.client.Request;
+import org.graylog.storage.elasticsearch7.PlainJsonApi;
+import org.graylog2.migrations.V20170607164210_MigrateReopenedIndicesToAliases;
+
+import javax.inject.Inject;
+import java.util.Collection;
+
+public class V20170607164210_MigrateReopenedIndicesToAliasesClusterStateES7 implements V20170607164210_MigrateReopenedIndicesToAliases.ClusterState {
+    private final PlainJsonApi plainJsonApi;
+
+    @Inject
+    public V20170607164210_MigrateReopenedIndicesToAliasesClusterStateES7(PlainJsonApi plainJsonApi) {
+        this.plainJsonApi = plainJsonApi;
+    }
+
+    @Override
+    public JsonNode getForIndices(Collection<String> indices) {
+        return plainJsonApi.perform(request(indices),
+                "Couldn't read cluster state for reopened indices " + indices);
+    }
+
+    private Request request(Collection<String> indices) {
+        final StringBuilder apiEndpoint = new StringBuilder("/_cluster/state/metadata");
```

Comment:
I think it would make sense to reuse `ClusterStateApi` here. The code in there is the exact same and i find the risk of unexpectedly breaking the migration low enough.